packages/upswyng-web/src/components/Hotlines.tsx

```diff
@@ -1,9 +1,101 @@
+    phone: "1-800-273-8255",
+  },
+  {
+    name: "National Child Abuse Hotline",
+    description: "",
+    //"The Childhelp National Child Abuse Hotline 1-800-4-A-CHILD (1-800-422-4453) is dedicated to the prevention of child abuse. Serving the United States, its territories, and Canada, the hotline is staffed 24 hours a day, 7 days a week with professional crisis counselors who, through interpreters, can provide assistance in over 170 languages. The hotline offers crisis intervention, information, literature, and referrals to thousands of emergency, social service, and support resources. All calls are confidential.",
+    phone: "1-800-422-4453",
+  },
+  {
+    name: "National Runaway Safeline",
+    description: "",
+    //"For Youth &amp; Teens, The National Runaway Safeline (NRS) is here to listen whether you are thinking of running away or already have. Our services are confidential and nonjudgmental. For Parents &amp;  Guardians, NRS can offer you support and help connect you to the right resources for your family. Whether you’re having trouble talking to your child or your child has already run, we can help. We will listen and assist you in developing a plan",
+    phone: "1-800-786-2929",
+  },
+  {
+    name: "GLBT National Hotline",
+    description: "",
+    phone: "1-800-843-4564",
+  },
+  {
+    name: "National Human Trafficking Resource Center",
+    description: "",
+    phone: "1-800-373-7888",
+  },
+  {
+    name: "National Sexual Assult Hotline",
+    description: "",
+    phone: "1-800-656-4673",
+  },
+  {
+    name: "Trans Lifeline",
+    description: "",
+    phone: "1-877-565-8860",
+  },
+  {
+    name: "Substance Abuse and Mental Health Services Administration",
+    description: "",
+    phone: "1-800-662-4357",
+  },
+  {
+    name: "National Hotline for Crime Victims",
+    description: "",
+    phone: "1-855-484-2846",
+  },
+  {
+    name: "United Way",
+    description: "",
+    phone: "2-1-1",
+  },
+];
+
+const hotlineCards = () =>
+  hotlineList.map((hotline, i) => (
+    <ListItem button key={i} onClick={() => initiatePhoneCall(hotline.phone)}>
+      <ListItemText
+        primary={hotline.name}
+        secondary={
+          <>
+            <Typography>{hotline.phone}</Typography>
+            {hotline.description}
+          </>
+        }
+      />
+    </ListItem>
+  ));
+
 const Hotlines = () => (
-  <div>
-    <p>Hotlines</p>
-  </div>
+  <Container>
+    <PageBanner color={colors.pink} text={"Hotlines"} />
+    <List>{hotlineCards()}</List>
```

Comment:
Personal pref, but I think you could throw that `hotlines.map` directly in the render 👍